dirty.html

```diff
@@ -1,49 +1,36 @@
+          <h2>General&nbsp;Assembly:&nbsp;Autumn/Winter&nbsp;2013</h2>
+          
           <div id="nav">
-            <div>
-              <div><link hef="#program">Program</link></div>
-              <div><link href="#students">Students</link></div>
-              <div><link hrf="#concepts">Concepts</link></div>
-              <div><link href="#instructors">Instructors</link></div>
-            </div>
-          </div>
-        </div>
-    </div>
-    <a name=program></a>
-    <div>
-      <div>
-        <div>
-          <div id="title" class="h3">The Program</div>
-          <a name="program"></link>
-          <div>              
-            <div>12 weeks of intensive web development training learning a range of programming techniques and languages.</d>
+            
+              <link href="#program">Program</link>
+              <link href="#students">Students</link>
+              <link href="#concepts">Concepts</link>
+              <link href="#instructors">Instructors</link>
           </div>
-        </div>
-        <div>
-          <div>
+    
+          <div id="title" 
+          <h3>The Program</h3>               
+            <div>12 weeks of intensive web development training learning a range of programming techniques and languages.</div>
+
             <div id="title">The Students</h5>
             <a name="students"></a>
             <div>The London WDI Session 3 Coding Collective</h3> 
-            <div>We faced missing routes, broken links, aborted rake tasks, rejected mass-assignments, failed deployments, nilClasses, illegal nestings, conflicted merges, rougue debuggers, unexpected keyword ends, incomplete divs, unintentional infinite loops, uncooperative asset pipelines, empty migrations, unresponsive APIs, undefined defined methods, overly-sassy css, dirty data, faulty gems, terrible animated gifs, excessive pizza, punchy classmates, demanding instructors and long days. And we won!</div>
+            <p>We faced missing routes, broken links, aborted rake tasks, rejected mass-assignments, failed deployments, nilClasses, illegal nestings, conflicted merges, rougue debuggers, unexpected keyword ends, incomplete divs, unintentional infinite loops, uncooperative asset pipelines, empty migrations, unresponsive APIs, undefined defined methods, overly-sassy css, dirty data, faulty gems, terrible animated gifs, excessive pizza, punchy classmates, demanding instructors and long days. And we won!</p>
           
           
-        </div>
-        <div >
+        
           
-          <div>
-            <div>The Concepts</div>
-            <a name="concepts"></div>
-            <div>We were taught a lot of concepts. Over 12 weeks with three projects, one hackathon and not much sleep, we learned and practiced a full range of front- and back-end concepts. Take a look at some of what we covered... </div>
+          
+            <h2>The Concepts</h2>
+            <div id="name" alt="concepts">
```

Comment:
`div` i believe doesn't have an `alt`, I think that's mainly for images (when they can't display)